infra/dss-events-scribe/main.tf

```diff
@@ -0,0 +1,76 @@
+data aws_caller_identity current {}
+data aws_region current {}
+
+locals {
+  region = "${data.aws_region.current.name}"
+  account_id = "${data.aws_caller_identity.current.account_id}"
+  common_tags = "${map(
+    "managedBy" , "terraform",
+    "Name"      , "${var.DSS_INFRA_TAG_SERVICE}-subscriptionsdynamodb",
+    "project"   , "${var.DSS_INFRA_TAG_PROJECT}",
+    "env"       , "${var.DSS_DEPLOYMENT_STAGE}",
+    "service"   , "${var.DSS_INFRA_TAG_SERVICE}",
+    "owner"     , "${var.DSS_INFRA_TAG_OWNER}"
+  )}"
+}
+
+locals {
+  replicas = ["aws", "gcp"]
+}
+
+data "aws_iam_policy_document" "sqs" {
+  statement {
+    principals {
+      type = "AWS"
+      identifiers = ["*"]
+    }
+    actions = ["sqs:SendMessage"]
+    resources = ["arn:aws:sqs:${local.region}:${local.account_id}:dss-events-scribe-${var.DSS_DEPLOYMENT_STAGE}"]
+    condition {
+      test     = "StringEquals"
+      variable = "aws:SourceArn"
+      values = ["${aws_cloudwatch_event_rule.events-scribe.arn}"]
+    }
+  }
+  statement {
+    principals {
+      type = "AWS"
+      identifiers = ["*"]
+    }
+    actions = [
+      "sqs:*",
+    ]
+    resources = ["arn:aws:sqs:${local.region}:${local.account_id}:dss-events-scribe-${var.DSS_DEPLOYMENT_STAGE}"]
+    condition {
+      test     = "StringEquals"
+      variable = "aws:SourceArn"
+      values   = ["arn:aws:lambda:${local.region}:${local.account_id}:function:dss-events-scribe-${var.DSS_DEPLOYMENT_STAGE}"]
+    }
+  }
+}
+
+resource "aws_sqs_queue" "dss-events-scribe-queue" {
+  name   					 = "dss-events-scribe-${var.DSS_DEPLOYMENT_STAGE}"
+  tags   					 = "${local.common_tags}"
+  message_retention_seconds  = "3600"
```

Comment:
What happens to the message after an hour?

Reply:
They're discarded.

Reply:
I believe that is answered by the documentation added in #2656: https://github.com/HumanCellAtlas/data-store/pull/2656/files#diff-431e856038c071e599fc68939e8d4365R18

> Messages older than `M > N` minutes are discarded from the queue